king_phisher/client/graphs.py

```diff
@@ -289,7 +311,6 @@ def refresh(self, info_cache=None, stop_event=None):
 		"""
 		Refresh the graph data by retrieving the information from the
 		remote server.
-
```

Comment:
I'd like to keep the blank lines between the documentation descriptions and parameters.